entrypoint.sh

```diff
@@ -84,11 +84,19 @@ settingsConfiguration() {
         fi
 
         if [ "$setting_type" = "plain" ]; then
-            echo "$setting_key = $setting_var" >> /healthchecks/hc/settings.py
+            if [ "$setting_key" = "ALLOWED_HOSTS" ]; then
+	        if [ "$setting_var" = "*" ]; then
+			echo "ALLOWED_HOSTS = [\"*\"]" >> /healthchecks/hc/local_settings.py
+		else
+			echo "$setting_key = [\"$setting_var\"]" >> /healthchecks/hc/local_settings.py
+		fi
+	    else 
+                echo "$setting_key = $setting_var" >> /healthchecks/hc/local_settings.py
+	    fi
```

Comment:
These changes above are not needed when specifying the allowed hosts as written above.